Mathlib/Topology/Algebra/UniformConvergence.lean

```diff
@@ -265,6 +265,11 @@ protected theorem UniformOnFun.hasBasis_nhds_one (𝔖 : Set <| Set α) (h𝔖
       { f : α →ᵤ[𝔖] G | ∀ x ∈ SV.1, f x ∈ SV.2 } :=
   UniformOnFun.hasBasis_nhds_one_of_basis 𝔖 h𝔖₁ h𝔖₂ (basis_sets _)
 
+@[to_additive (attr := simp)]
+lemma UniformOnFun.ofFun_prod {β : Type*} [CommMonoid β] {f : ι → α → β} (I : Finset ι) :
+    ∏ i ∈ I, ofFun 𝔖 (f i) = ofFun 𝔖 (∏ i ∈ I, f i) :=
+  rfl
```

Comment:
The name suggests that this should point the other way, as does [UniformFun.ofFun_add](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Topology/Algebra/UniformConvergence.html#UniformFun.ofFun_add)